examples/article.amp.html

```diff
@@ -162,7 +162,7 @@
             srcset="img/hero@1x.jpg 1x, img/hero@2x.jpg 2x"
             layout="responsive" width="360" placeholder
             alt="Curabitur convallis, urna quis pulvinar feugiat, purus diam posuere turpis, sit amet tincidunt purus justo et mi."
-            height="216" on="tap:headline-img-lightbox">
+            height="216" on="tap:headline-img-lightbox" role="img" tabindex="0">
```

Comment:
It's not because it is an amp-img, but because of the attribute 'on' having a 'tap:...' value. We've concluded that some tags such as 'a' and 'button' do have an implicit role and tabindex. Do you believe tags 'img' and 'amp-img' should as well?